torch/csrc/utils/tensor_new.cpp

```diff
@@ -590,10 +596,10 @@ Tensor new_tensor(const Type& type, ScalarType scalar_type, PyObject* args, PyOb
 
 Tensor new_empty(const Type& type, ScalarType scalar_type, PyObject* args, PyObject* kwargs) {
   static PythonArgParser parser({
-    "new_empty(IntArrayRef size, *, ScalarType dtype=None, Device? device=None, bool requires_grad=False)",
+    "new_empty(IntArrayRef size, *, ScalarType dtype=None, Device? device=None, bool pin_memory=False, bool requires_grad=False)",
```

Comment:
what's the story here?  Don't the parameter indices need to be updated here (and why don't the rest of the functions below have the parameter?).

Reply:
Nice catch! I think we already had discussion about named parameters in PythonArgParser, right?